packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -529,26 +525,107 @@ class RenderListWheelViewport
     size = constraints.biggest;
   }
 
+  /// Get the index of a child by looking at its parentData.
+  int indexOf(RenderBox child) {
+    assert(child != null);
+    final ListWheelParentData childParentData = child.parentData;
+    assert(childParentData.index != null);
+    return childParentData.index;
+  }
+
+  /// Return the index of the child that should be at the given offset.
+  int offsetToIndex(double offset) => offset ~/ itemExtent;
```

Comment:
'offset' is too overloaded as a term. Call it scrollOffset here and below.

Reply:
done.